CONTRIBUTING.md

```diff
@@ -410,13 +410,18 @@ Frontend assets (TypeScript, JavaScript, CSS, and images) must be compiled in or
 
 #### nvm and node
 
-First, be sure you are using recent versions of NodeJS and npm. Using [nvm](https://github.com/creationix/nvm) to manage them is recommended. Check the docs at the link to be sure, but at the time of writing the following would install nvm and node:
+First, be sure you are using recent versions of NodeJS and npm. We recommend using [nvm](https://github.com/nvm-sh/nvm) to manage your node environment:
 
 ```bash
-curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | bash
-nvm install node
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.0/install.sh | bash
+nvm install v12
+
+cd superset-frontend
+nvm use
```

Comment:
can we rework this so the version isn't hard-coded, maybe 

```suggestion

cd superset-frontend
nvm use
nvm install
```

Reply:
as per https://github.com/nvm-sh/nvm#nvmrc once the directory has an `.nvmrc` file it'll always reference that version when left blank